README.Rmd

```diff
@@ -70,30 +70,30 @@ create_irods("http://localhost/irods-rest/0.9.3", "/tempZone/home")
 # authenticate
 iauth("rods", "rods")
 
-# add user bobby
-rirods:::iadmin(action = "add", target = "user", arg2 = "bobby", arg3 = "rodsuser")
+# add user Alice
+rirods:::iadmin(action = "add", target = "user", arg2 = "alice", arg3 = "rodsuser")
 
-# modify bobby's password
-rirods:::iadmin(action = "modify", target = "user", arg2 = "bobby", arg3 = "password", arg4  = "passWORD")
+# modify Alice's password
+rirods:::iadmin(action = "modify", target = "user", arg2 = "alice", arg3 = "password", arg4  = "passWORD")
 ```
 
 ### authentication
 
-In this example Bobby is a user of iRODS and he can authenticate himself with `iauth()`. This prompts a dialog where you can enter your username and password without hardcoding this information in your scripts.
+In this example Alice is a user of iRODS and he can authenticate himself with `iauth()`. This prompts a dialog where you can enter your username and password without hardcoding this information in your scripts.
```

Comment:
I would suggest using 'rods' as username and password, since that way someone who is trying this out in the demo API (like I was a few months ago) and has no username of their own can test things.

Reply:
Maybe we can improve the non-exported `use_irods_demo()` helper function which launches the docker (and creates the user Alice in a future extension). And then suggest user/developers to use that function instead. Might be good to have an appealing story-like documentation, also for future workshops and such.

Reply:
Ohh that would be nice!